templates/contest/contest.html

```diff
@@ -27,7 +27,8 @@
 {% block body %}
     <div id="banner">
         <a href="https://www.timeanddate.com/worldclock/fixedtime.html?msg={{ contest.name|urlquote('') }}&amp;iso=
-                {{- contest.start_time|utc|date('Y-m-d\TH:i:s') }}" class="date">
+                {% if contest.start_time > now %} {{- contest.start_time|utc|date('Y-m-d\TH:i:s') }}
```

Comment:
I think we can write this using a ternary, with

```
{{ (... if ... else ...)|utc|... }}
```

so that we don't have to write the `|utc|date...` filter twice.

Reply:
Also a small nit: commit names should be descriptive about what you are fixing, e.g. "Make contest timeanddate link to end time after contest has started".